.github/scripts/parse_rpt.py

```diff
@@ -1,33 +1,66 @@
 import sys
+import json
+import os
+import re, subprocess
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from common.check_gen_files import check_gen_files
 
 sys.stdout.flush()
 
-if len(sys.argv) == 1 or sys.argv[1] == "sky130hvl_ldo":
+if (len(sys.argv) > 1) and (sys.argv[1] == "sky130hd_cryo"):
+    drc_filename = "flow/reports/sky130hd/cryo/6_final_drc.rpt"
+    lvs_filename = "flow/reports/sky130hd/cyro/6_final_lvs.rpt"
+elif (len(sys.argv) == 1) or (sys.argv[1] == "sky130hvl_ldo"):
     drc_filename = "work/6_final_drc.rpt"
     lvs_filename = "work/6_final_lvs.rpt"
 else:
     drc_filename = "work/"+sys.argv[1]+"/6_final_drc.rpt"
     lvs_filename = "work/"+sys.argv[1]+"/6_final_lvs.rpt"
 
-if len(sys.argv) > 1 and sys.argv[1] == "sky130hvl_ldo":
+# using full because sims disabled for LDO for now
+if (len(sys.argv) > 1) and ((sys.argv[1] == "sky130hvl_ldo") or (sys.argv[1] == "sky130hvl_ldo_full")):
     with open(drc_filename, 'r') as f1, open("../../../.github/scripts/expected_drc_reports/expected_ldo_drc.rpt", 'r') as f2:
         content1 = f1.readlines()
         content2 = f2.readlines()
-        if content1 == content1:
+        if content1 == content2:
             print("DRC is clean!")
         else:
             raise ValueError("DRC failed!")
-        
 elif sum(1 for line in open(drc_filename)) > 3:
     raise ValueError("DRC failed!")
 else:
     print("DRC is clean!")
 
-
-with open(lvs_filename) as f:
-    f1 = f.read()
-
-    if "failed" in f1:
+# cryo LVS check 
+if (len(sys.argv) > 1) and (sys.argv[1] == "sky130hd_cryo"):
+    lvs_line = subprocess.check_output(["tail", "-1", lvs_filename]).decode(
+        sys.stdout.encoding
+    )
+    regex = r"failed"
+    match = re.search(regex, lvs_line)
+    
+    if match != None:
         raise ValueError("LVS failed!")
     else:
         print("LVS is clean!")
+else:        
+    with open(lvs_filename) as f:
+        f1 = f.read()
+    
+        if "failed" in f1:
+            raise ValueError("LVS failed!")
+        else:
+            print("LVS is clean!")
+
+if ((len(sys.argv) > 1) and ((sys.argv[1] == "sky130hvl_ldo") or (sys.argv[1] == "sky130hvl_ldo_full"))) or ((len(sys.argv) > 1) and (sys.argv[1] == "sky130hd_cryo")):
```

Comment:
@chetanyagoyal : can you please explain what you are trying to do here? I don't understand what you are expecting as arguments to this script.

Reply:
Sure! Its based off the way parse_rpt.py is being called in the respective yaml files 
For ex. - if its `python3 parse_rpt.py sky130hvl_ldo`, I want this statement to be triggered and argv[1] here will be sky130hvl_ldo.

Reply:
ok but why you don't want to check the generated files using `check_gen_files()` if these (`sky130hvl_ldo`, `sky130hvl_ldo_full`, `sky130hd_cryo` are the arguments, and consider directly that the flow is clean?

Reply:
Oh right, I have already removed this statement. I meant to do that earlier but it slipped my mind. These generators will also be checked with the new script